res/database/Default/user/cujo.xml

```diff
@@ -0,0 +1,1807 @@
+        </news>
+		<!-- "TVT AG" -->
+        <news id="cujo-news-tvt-01" type="0" thread_id="cujo-news-tvt" creator="8827" created_by="Cujo">
+            <title>
+                <de>TVT-Entwickler arbeiten im Home-Office</de>
+                <en>TVT developers work in a home office</en>
+            </title>
+            <description>
+                <de>Wegen der Pandemie arbeiten ab sofort alle TVT-Entwickler im Home-Office. Laut der TVT AG sind weltweit etwa 35.000 Mitarbeiter davon betroffen.</de>
+                <en>Due to the pandemic, all TVT developers are working in their home offices with immediate effect. According to TVT AG, about 35,000 employees worldwide are affected.</en>
+            </description>
+			<effects>
+                <effect trigger="happen" type="triggernews" time="2,1,2,12,16" news="cujo-news-tvt-02" />
+            </effects>
+            <data genre="0" price="0.5" quality="20" />
+            <availability year_range_from="2020" year_range_to="" />
+        </news>
+        <news id="cujo-news-tvt-02" type="2" thread_id="cujo-news-tvt" creator="8827" created_by="Cujo">
+            <title>
+                <de>News werden verschlüsselt</de>
+                <en>News are encrypted</en>
+            </title>
+            <description>
+                <de>Laut der TVT AG werden die News wegen der geltenden GSDVO ab sofort verschlüsselt. v/KNne yTZbUgc+p FMJ6 MH8qT XQFrZsi.</de>
+                <en>According to TVT AG, the news will be encrypted with immediate effect due to the GSDVO in force. v/KNne yTZbUgc+p FMJ6 MH8qT XQFrZsi.</en>
+            </description>
+			<effects>
+                <effect trigger="happen" type="triggernews" time="2,1,2,13,17" news="cujo-news-tvt-03" />
+            </effects>
+            <data genre="0" price="0.5" quality="30" />
+        </news>
+        <news id="cujo-news-tvt-03" type="2" thread_id="cujo-news-tvt" creator="8827" created_by="Cujo">
+            <title>
+                <de>TVT AG schafft Schreibmaschinen ab</de>
+                <en>TVT AG abolishes typewriters</en>
+            </title>
+            <description>
+                <de>Wie die Pressesprecherin der TVT AG mitteilte, müssen die Entwickler nicht mehr auf Schreibmaschinen programmieren: "Unsere Programmierer werden ab sofort mit leistungsfähigen 16-Bit-Bandium-Computern ausgestattet."</de>
+                <en>According to TVT AG's press spokeswoman, developers no longer have to program on typewriters: "Our programmers will be equipped with powerful 16-bit Bandium computers from now on."</en>
+            </description>
+			<effects>
+                <effect trigger="happen" type="triggernews" time="2,1,2,14,18" news="cujo-news-tvt-04" />
+            </effects>
+            <data genre="0" price="0.5" quality="30" />
+        </news>
+        <news id="cujo-news-tvt-04" type="2" thread_id="cujo-news-tvt" creator="8827" created_by="Cujo">

```

Comment:
Gar kein Zusammenhang mit TVT

Reply:
Das Interview wurde in einem TVT-Sender geführt. Da ich den Sendernamen nicht genannt habe, passt er für alle.